discovery-provider/src/api/v1/models/playlists.py

```diff
@@ -31,10 +30,14 @@
         "id": fields.String(required=True),
         "is_album": fields.Boolean(required=True),
         "playlist_name": fields.String(required=True),
+        "playlist_contents": fields.List(
+            fields.Nested(playlist_added_timestamp), required=True
+        ),
         "repost_count": fields.Integer(required=True),
         "favorite_count": fields.Integer(required=True),
         "total_play_count": fields.Integer(required=True),
         "user": fields.Nested(user_model, required=True),
+        "cover_art_sizes": fields.String,
```

Comment:
Kinda stinks that we need this, as it makes the V1 api less approachable and is due to not being able to selectively update without the full metadata each time (wish we could just pass undefined for things we don't want to change 😢 ), but I see why you added it. I wonder if we need a quick thumbs up from Forrest since we're touching the public facing version of the API.

Reply:
100% that this update path is more difficult than it needs to be. Let me look a little deeper into EM to see if it's possible to do selective updates.

I would argue that having these fields doesn't necessarily make the api less approachable though, having the image and playlist contents in the non-full endpoint seems correct to me

Reply:
Yeah having the cover art makes sense, but `cover_art_sizes` is a CID not the URL(s) to the cover art, right? This is the kind of implementation detail that V1 typically tries to avoid imo but that's not to say we shouldn't do it

Reply:
Oh right, so ideally we would be returning an object of paths? And then content node selection still happens on client side or would the urls be fully qualified server side?

Reply:
Using `v1/users` as an example, it would be consistent to have the full paths in an object keyed by size: https://discoveryprovider.audius.co/v1/users/Avx7v

I wonder if we're going to run into similar issues with every non-full entity? Like does updating a user without specifying `cover_photo_sizes` have the same problem? Maybe we only allow updates to v1/full entities? Or we'll have to decide to include the CIDs in non-full responses for _all_ entities?

There's a bit of coupling right now (even prior to your PR or entity manager) with the response given back from DN for an entity, and the metadata that we send to create/update an entity. Arguably these should be separate things with separate models and separate validation, and we shouldn't be fetching the response for an entity and then passing it into a request to update it. The data we send for creates and updates could be different - some fields might not be editable. There's probably derived data on responses, like aggregations etc that would be ignored. Or this case, where we want pretty looking responses but need the core guts for updates thanks to the coupling. But we're in kinda deep now, and shaking this pattern seems hard.